src/modules/highlight/renderer/common/cssselector.ts

```diff
@@ -27,11 +27,11 @@ export const fullQualifiedSelector = (
   justSelector: boolean
 ): string => {
   if (node.nodeType !== Node.ELEMENT_NODE) {
-    const lowerCaseName =
+    return (
       ((node as Element).localName &&
         (node as Element).localName.toLowerCase()) ||
-      node.nodeName.toLowerCase();
-    return lowerCaseName;
+      node.nodeName.toLowerCase()
+    );
```

Comment:
One way to write this that might be easier to read is:
```
return (node as Element)?.localName.toLowerCase() ?? node.nodeName.toLowerCase()
```
where you use optional chaining plus [nullish coalescing](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html) to provide a fallback in case it ends up being undefined somewhere

Reply:
I'll double check this as well tomorrow, this was also a result of accepting suggested lint ;)

Reply:
i'll use your recommendation, this entire function is currently not even used, but i didn't want to delete it yet before i double check if we need it